public/app/features/integrated-alerting/components/AlertRules/AlertRulesTable/AlertRulesTable.tsx

```diff
@@ -0,0 +1,157 @@
+import { getStyles } from './AlertRulesTable.styles';
+import { css } from 'emotion';
+import { logger } from '@percona/platform-core';
+import { AlertRulesService } from '../AlertRules.service';
+import { Messages } from '../../../IntegratedAlerting.messages';
+import { AlertRule } from '../AlertRules.types';
+import { formatRules } from './AlertRulesTable.utils';
+
+const { noData, columns } = Messages.alertRules.table;
+
+const {
+  createdAt: createdAtColumn,
+  duration: durationColumn,
+  filters: filtersColumn,
+  lastNotified: lastNotifiedColumn,
+  severity: severityColumn,
+  summary: summaryColumn,
+  threshold: thresholdColumn,
+} = columns;
+
+export const AlertRulesTable = () => {
+  const style = useStyles(getStyles);
+  const [pendingRequest, setPendingRequest] = useState(false);
+  const [data, setData] = useState<AlertRule[]>([]);
+
+  const getAlertRules = async () => {
+    setPendingRequest(true);
+    try {
+      const { rules } = await AlertRulesService.list();
+      setData(formatRules(rules));
+    } catch (e) {
+      logger.error(e);
+    } finally {
+      setPendingRequest(false);
+    }
+  };
+
+  const columns = React.useMemo(
+    () => [
+      {
+        Header: summaryColumn,
+        accessor: 'summary',
+        width: '25%',
+      } as Column,
+      {
+        Header: thresholdColumn,
+        accessor: 'threshold',
+        width: '10%',
+      } as Column,
+      {
+        Header: durationColumn,
+        accessor: 'duration',
+        width: '10%',
+      } as Column,
+      {
+        Header: severityColumn,
+        accessor: 'severity',
+        width: '5%',
+      } as Column,
+      {
+        Header: filtersColumn,
+        accessor: ({ filters }: AlertRule) => (
+          <div className={style.filtersWrapper}>
+            {filters.map(filter => (
+              <span key={filter} className={style.filter}>
+                {filter}
+              </span>
+            ))}
+          </div>
+        ),
+        width: '30%',
+      } as Column,
+      {
+        Header: createdAtColumn,
+        accessor: 'createdAt',
+        width: '10%',
+      } as Column,
+      {
+        Header: lastNotifiedColumn,
+        accessor: ({ lastNotified }: AlertRule) => (
+          <>
+            <div className={style.lastNotifiedWrapper}>
+              {lastNotified ? (
+                <>
+                  <span className={style.lastNotifiedDate}>{lastNotified}</span>
+                  <span className={style.lastNotifiedCircle} />
+                </>
+              ) : null}
+            </div>
+          </>
+        ),
+        width: '10%',
+      } as Column,
+    ],
+    []
+  );
+
+  useEffect(() => {
+    getAlertRules();
+  }, []);
+
+  const tableInstance = useTable({ columns, data });
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } = tableInstance;
+
+  return (
+    <div className={style.tableWrap} data-qa="alert-rules-table-outer-wrapper">
```

Comment:
Created a Table component in Notification Channels list task so we won't need to repeat this code and the styles and it will be easier to refactor (just in one place), when we want to change the table. So maybe next sprint we can do some refactor and use it here :)

Reply:
Thanks Tiago, ok, but let's discuss about that, since we actually might want to keep similar pieces of codes for more flexibility and ease of customization vs the overhead of creating a common, yet to be customizable, codebase. We had a discussion on that recently and agreed that duplication, at least in initial phases of development, can be better. Let's move these improvements to later stages of development :)